cicd/forgeops-tests/forgeops-tests.py

```diff
@@ -28,6 +68,8 @@
     html_report_path = os.path.join(report_path, html_report_name)
     allure_report_path = os.path.join(report_path, 'allure-files')
 
+    set_allure_environment_props(os.path.join(allure_report_path, 'environment.properties'))
```

Comment:
On my machine the allure report does not contains a .properties file but a widgets/environment.json.
Maybe on new version of allure the property file has been replaced?

Reply:
Allure's docs, https://docs.qameta.io/allure/#_environment, states "...just create environment.properties (or environment.xml)..."

Reply:
```
TESTS_NAMESPACE=markg
TESTS_JAVA_VERSION=1.8.0_121
TESTS_DOMAIN=forgeops.com
```
Note:  the file will be created based upon the environment variables passed in starting with "TEST_".